modules/nf-core/tcoffee/align/tests/main.nf.test

```diff
@@ -0,0 +1,119 @@
+nextflow_process {
+
+    name "Test Process TCOFFEE_ALIGN"
+    script "../main.nf"
+    process "TCOFFEE_ALIGN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tcoffee"
+    tag "tcoffee/align"
+
+    test("fasta - align_sequence") {
+
+        config "./sequence.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ],
+                             file("https://raw.githubusercontent.com/nf-core/test-datasets/multiplesequencealign/testdata/setoxin-ref.fa", checkIfExists: true)
+                           ]
+                input[1] = [[:],[]]
+                input[2] = [[:],[],[]]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.msa).match("msa")},
```

Comment:
based on the meta.yml, shouldn' it be `process.out.alignment`?

Reply:
Good catch, in the `main` was `msa` but `alignment` will be more consistent with the other MSA modules, so I changed it now.